### ISSUE TITLE
[Bug]: toast doesn't show after redirect

### ISSUE THREAD

**Issue by abedcodes** [tasteui/tasteui#409]
### What Happened

Hello AJ, Thanks for this amazing library of components. 
Toast works if you stay on the same page but not if you redirect afterward!
what to do with this?

```
$this->toast()->success('welcome')->send();
$this->redirect(route('admin.dashboard'));
```

### How to Reproduce

.

### Laravel Version

10.4

### Livewire Version

3.4

### TallStackUI Version

1.2

### PHP Version

8.2

### Operation System

Windows

**Comment by devajmeireles** [comment]
Done! Waiting for the next release. Thanks, @abedcodes .

PR: https://github.com/tallstackui/tallstackui/pull/416

@MGeurts, take a look into this.

**Comment by MGeurts** [comment]
Will test and use it as soon as being released.

Thanks AJ.
We realy appreciate the hard work.

**Comment by devajmeireles** [comment]
The feature was published in the latest release: v1.23.0. Please, upgrade your project:
https://tallstackui.com/docs/interaction/dialog#display-after-redirects
